StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/STPAPIClient+Link.swift

```diff
@@ -370,7 +370,7 @@ extension STPAPIClient {
         cvc: String?,
         expectedPaymentMethodType: String?,
         billingPhoneNumber: String?,
-        clientAttributionMetadata: STPClientAttributionMetadata,
+        clientAttributionMetadata: STPClientAttributionMetadata?,
```

Comment:
nit: this is no longer considered a "refactor" as we are fixing an issue here.

It's a bit confusing when we should/shouldn't pass `clientAttributionMetadata` -- we should probably comment this at the call site of sharePaymentDetails comment out why we are passing 'nil' vs not nil.